[2022-07-05 07:21:41,230] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [queued]>
[2022-07-05 07:21:41,257] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [queued]>
[2022-07-05 07:21:41,257] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:41,257] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:21:41,257] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:21:41,298] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 06:00:00+00:00
[2022-07-05 07:21:41,304] {standard_task_runner.py:52} INFO - Started process 611 to run task
[2022-07-05 07:21:41,315] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T06:00:00+00:00', '--job-id', '259', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpwctt_f9s', '--error-file', '/tmp/tmpedloueil']
[2022-07-05 07:21:41,316] {standard_task_runner.py:77} INFO - Job 259: Subtask kimuksung_test
[2022-07-05 07:21:41,450] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:21:41,582] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T06:00:00+00:00
[2022-07-05 07:21:41,583] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:21:41,584] {taskinstance.py:1718} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:41,604] {taskinstance.py:1282} INFO - Marking task as FAILED. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T060000, start_date=20220705T072141, end_date=20220705T072141
[2022-07-05 07:21:41,628] {standard_task_runner.py:92} ERROR - Failed to execute job 259 for task kimuksung_test
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1334, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1460, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1516, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 174, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 188, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in test
    parameter=context["paramas"]["parameter1"]
KeyError: 'paramas'
[2022-07-05 07:21:41,648] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-07-05 07:21:41,704] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 07:25:54,881] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [queued]>
[2022-07-05 07:25:54,901] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [queued]>
[2022-07-05 07:25:54,901] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:54,901] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 07:25:54,901] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 07:25:54,926] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 06:00:00+00:00
[2022-07-05 07:25:54,940] {standard_task_runner.py:52} INFO - Started process 802 to run task
[2022-07-05 07:25:54,955] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T06:00:00+00:00', '--job-id', '290', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp0x0_p3f6', '--error-file', '/tmp/tmpq7kaafde']
[2022-07-05 07:25:54,956] {standard_task_runner.py:77} INFO - Job 290: Subtask kimuksung_test
[2022-07-05 07:25:55,095] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 07:25:55,200] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T06:00:00+00:00
[2022-07-05 07:25:55,201] {logging_mixin.py:109} INFO - test function start
[2022-07-05 07:25:55,202] {test.py:22} INFO - pubg
[2022-07-05 07:25:55,202] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 07:25:55,233] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T060000, start_date=20220705T072554, end_date=20220705T072555
[2022-07-05 07:25:55,291] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 07:25:55,329] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:06:35,006] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [queued]>
[2022-07-05 08:06:35,044] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [queued]>
[2022-07-05 08:06:35,044] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:35,044] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:06:35,044] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:06:35,094] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 06:00:00+00:00
[2022-07-05 08:06:35,100] {standard_task_runner.py:52} INFO - Started process 2202 to run task
[2022-07-05 08:06:35,117] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T06:00:00+00:00', '--job-id', '349', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpm0fm_kk8', '--error-file', '/tmp/tmpfowsctgy']
[2022-07-05 08:06:35,119] {standard_task_runner.py:77} INFO - Job 349: Subtask kimuksung_test
[2022-07-05 08:06:35,347] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:06:35,735] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T06:00:00+00:00
[2022-07-05 08:06:35,737] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:06:35,738] {test.py:22} INFO - pubg
[2022-07-05 08:06:35,738] {python.py:175} INFO - Done. Returned value was: test function clear
[2022-07-05 08:06:35,939] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T060000, start_date=20220705T080635, end_date=20220705T080635
[2022-07-05 08:06:36,068] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:06:36,218] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-07-05 08:11:51,872] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [queued]>
[2022-07-05 08:11:51,922] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [queued]>
[2022-07-05 08:11:51,922] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:51,922] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-07-05 08:11:51,922] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-07-05 08:11:52,033] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): kimuksung_test> on 2022-07-05 06:00:00+00:00
[2022-07-05 08:11:52,109] {standard_task_runner.py:52} INFO - Started process 2508 to run task
[2022-07-05 08:11:52,123] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'uk_dag_test_1', 'kimuksung_test', 'scheduled__2022-07-05T06:00:00+00:00', '--job-id', '416', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpsvz1d9zd', '--error-file', '/tmp/tmparf5srn9']
[2022-07-05 08:11:52,124] {standard_task_runner.py:77} INFO - Job 416: Subtask kimuksung_test
[2022-07-05 08:11:52,693] {logging_mixin.py:109} INFO - Running <TaskInstance: uk_dag_test_1.kimuksung_test scheduled__2022-07-05T06:00:00+00:00 [running]> on host 9514ca4c721b
[2022-07-05 08:11:53,050] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=uk_dag_test_1
AIRFLOW_CTX_TASK_ID=kimuksung_test
AIRFLOW_CTX_EXECUTION_DATE=2022-07-05T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-05T06:00:00+00:00
[2022-07-05 08:11:53,051] {logging_mixin.py:109} INFO - test function start
[2022-07-05 08:11:53,052] {test.py:22} INFO - pubg
[2022-07-05 08:11:53,052] {python.py:175} INFO - Done. Returned value was: pubg
[2022-07-05 08:11:53,156] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=uk_dag_test_1, task_id=kimuksung_test, execution_date=20220705T060000, start_date=20220705T081151, end_date=20220705T081153
[2022-07-05 08:11:53,269] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-05 08:11:53,438] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
